<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(11, 0, 57);</string>
  </property>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loginView_btn_login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginView_feedback">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(11, 0, 57);color:rgb(255, 0, 0);font-size:12pt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginView_email">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginView_password">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 157, 255);color:rgb(212, 255, 70);font-size:18pt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
